src/main/java/org/embulk/input/google_ads/GoogleAdsReporter.java

```diff
@@ -223,7 +222,6 @@ public SearchGoogleAdsRequest buildRequest(PluginTask task, String query)
     {
         return SearchGoogleAdsRequest.newBuilder()
                 .setCustomerId(task.getCustomerId())
-                .setPageSize(PAGE_SIZE)
```

Comment:
https://ads-developers.googleblog.com/2024/07/upcoming-changes-to-page-size-in-google.html
> What do I need to change?
If you use version v17 of the API, no changes are required. If you use an older version of the API and set the page_size field in your [GoogleAdsService.Search](https://developers.google.com/google-ads/api/reference/rpc/v17/GoogleAdsService#search) requests, then you should update your application code to stop setting this field.